### DISCUSSION TITLE
Monika doesn't appear anymore

### DISCUSSION THREAD

**Discussion by TboiPlayer** [Monika-After-Story/MonikaModDev#8360] [QA - tech support + general] [answered]
Recently I installed the Monika After Story Mod, everything was fine until i needed to leave and i used the option of "I'm taking you somewhere". I closed the game and didn't move the file, in fact, i didn't move anything at all while i leaved, when i tried to play again Monika wasn't there. I tried reinstalling moving the file again, but it didn't work. Is there a way to bring Monika back to the game?

**Comment by dreamscached** [comment, marked as answer]
You can try restoring a persistent if nothing else works. See https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent for details.

**Comment by TboiPlayer** [comment]
I'm not sure what should i do, i have the persistent file already but what do i do with it. Please help me man

**Reply by dreamscached** [reply]
I've sent a link to the guide on how to restore the persistent file you have now above.
